src/containers/pages/InterventionPlan/NewPlan/General/tests/index.test.tsx

```diff
@@ -217,20 +232,20 @@ describe('containers/pages/NewPlan', () => {
         .props() as any).formik.values
     );
 
-    expect(mockedStore.dispatch).toHaveBeenLastCalledWith(addPlanDefinition(payload));
+    expect(mockedStore.dispatch).toHaveBeenCalledWith(addPlanDefinition(payload));
```

Comment:
Why does this change?  I'm trying to understand how this ensures that we are testing for the feature that you added.

Reply:
The order of dispatch changes. as the mocked store initial dispatches in initial state into the store itself thats why instead of testing for the last dispatch we can arbitrary check for weather it is called or not.